File: codegen/results/v001/qwen3.5_27b_think/20260408_122500/go/002_word_frequency/solution.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"sort"
	"strings"
	"unicode"
)

func main() {
	file, err := os.Open("input/text.txt")
	if err != nil {
		fmt.Fprintln(os.Stderr, err)
		os.Exit(1)
	}
	defer file.Close()

	wordCount := make(map[string]int)

	scanner := bufio.NewScanner(file)
	for scanner.Scan() {
		line := scanner.Text()
		cleanLine := strings.Map(func(r rune) rune {
			if unicode.IsLetter(r) {
				return unicode.ToLower(r)
			}
			return ' '
		}, line)

		words := strings.Fields(cleanLine)
		for _, word := range words {
			if word != "" {
				wordCount[word]++
			}
		}
	}

	if err := scanner.Err(); err != nil {
		fmt.Fprintln(os.Stderr, err)
		os.Exit(1)
	}

	type entry struct {
		word  string
		count int
	}
	var entries []entry
	for word, count := range wordCount {
		entries = append(entries, entry{word, count})
	}

	sort.Slice(entries, func(i, j int) bool {
		if entries[i].count != entries[j].count {
			return entries[i].count > entries[j].count
		}
		return entries[i].word < entries[j].word
	})

	for _, e := range entries {
		fmt.Printf("%s: %d\n", e.word, e.count)
	}
}

File: codegen/results/v001/qwen3.5_27b_think/20260408_122500/typescript/002_word_frequency/solution.ts
import * as fs from 'fs';

const filePath = 'input/text.txt';
const content = fs.readFileSync(filePath, 'utf-8');

const words = content.toLowerCase().match(/[a-z]+/g) || [];

const wordCount: { [word: string]: number } = {};

for (const word of words) {
    wordCount[word] = (wordCount[word] || 0) + 1;
}

const sorted = Object.entries(wordCount).sort((a, b) => {
    if (b[1] !== a[1]) {
        return b[1] - a[1];
    }
    return a[0].localeCompare(b[0]);
});

for (const [word, count] of sorted) {
    console.log(`${word}: ${count}`);
}